src/topology/server.rs

```diff
@@ -5,7 +5,7 @@ use Error::{self, OperationError};
 use bson::oid;
 use connstring::Host;
 use pool::{ConnectionPool, PooledStream};
-use stream::StreamConnector;
+use stream::ConnectMethod;
```

Comment:
Changes to this file are a mix of the renaming of `StreamConnector` and from rustfmt.